util/src/main/java/io/kubernetes/client/Attach.java

```diff
@@ -57,18 +57,14 @@ public void setApiClient(ApiClient apiClient) {
     this.apiClient = apiClient;
   }
 
-  private String makePath(
-      String namespace, String name, String container, boolean stdin, boolean tty) {
-    return "/api/v1/namespaces/"
-        + namespace
-        + "/pods/"
-        + name
-        + "/attach?"
-        + "stdin="
-        + stdin
-        + "&tty="
-        + tty
-        + (container != null ? "&container=" + container : "");
+  /**
+   * Setup a Builder for the given namespace and name
+   *
+   * @param namespace The namespace of the Pod
+   * @param name The name of the Pod
+   */
+  public Builder prepare(String namespace, String name) {

```

Comment:
Perhaps `newBuilder()` or `getBuilder()` or just `builder()` instead of `prepare()`?

Reply:
I spent a long time pondering the naming here. It's not really a "builder" per-se, it more of a configured connector of sorts. But I guess `builder()` is good enough.

Reply:
`Attach` is more of a client class so the name `newBuilder()` sounds like it's meant to build another deep copy of the client, which is not the case. i'd vote on `attach.newConnectionBuilder()...build()`.

Reply:
or maybe even make it a static on the `Builder` class itself? (does that work in Java?) so it would be:

`attach.Builder.newBuilder()...build()`

Reply:
I don't think you can use "new" for a method name (reserved keyword). I've changed the code to "newConnectionBuilder()" vs. "newExecutionBuilder".